desktop/core/src/desktop/static/desktop/js/admin-wizard-inline.js

```diff
@@ -168,10 +168,10 @@ $(document).ready(function () {
     });
   });
 
-  $("#updateSkipWizard").prop('checked', $.cookie("hueLandingPage", { path: "/" }) == "home");
+  $("#updateSkipWizard").prop('checked', Cookies.set("hueLandingPage", { path: "/" }) == "home");
```

Comment:
This looks incorrect, but so was the old code even before the inline js migration. Lets leave it for this PR.